packages/blockchain-wallet-v4-frontend/src/data/coins/utils.ts

```diff
@@ -41,11 +41,16 @@ export const generateInterestAccount = (
 
 export const generateProvisionalPaymentAmount = (
   coin: CoinType,
-  amount: number
+  amount: number,
+  orgCoin?: CoinType
 ): string | number => {
   if (coin === 'BTC' || coin === 'BCH') {
     return parseInt(convertStandardToBase(coin, amount))
   }
 
+  if (coin === 'ETH' && orgCoin && orgCoin === 'USDT') {
+    return parseInt(convertStandardToBase(orgCoin, amount))
+  }
```

Comment:
this cover USDT case, but we might create a more generic solution if needed in future, anyhow USDT is a special case.